style.css

```diff
@@ -1996,6 +2001,12 @@ body.template-cover .entry-header {
 
 /* Post Footer ------------------------------- */
 
+/* READ MORE LINK */
+
+a.more-link {
+	margin-right: 1rem;
+}
```

Comment:
I think this should be the only change for this PR?
It might make more sense to put the margin on the Edit link, since it appears in several different places.

Reply:
I thought about that as well, but the space should only occur if the EDIT link comes behind another link such as the READ MORE link. By adding the space to the EDIT button it would always show a space before that link, even if the EDIT link is the only link visible. Thus, this would lead to a slight offset of the link. Do you know what I mean?

Reply:
I do know, and since it's only shown to logged in users, I think it doesn't matter so much.

Reply:
Yeah, that’s a good point! I’ll adjust the PR accordingly.